Tests/DatadogTests/Datadog/Logging/LoggingMessageReceiverTests.swift

```diff
@@ -161,17 +161,23 @@ class LoggingMessageReceiverTests: XCTestCase {
         )
 
         // When
-        let sent = Event(test: .mockRandom())
```

Comment:
Cleanup/ we can now remove the `Event` declaration above:
```swift
struct Event: Encodable {
   let test: String
}
```

Reply:
Ah good catch!! 👀